ansible/roles/puma/tasks/main.yml

```diff
@@ -21,6 +21,11 @@
 - name: Clean up old puma monit scripts
   shell: rm -f /etc/monit/conf.d/puma_*
 
+- name: mkdir /etc/monit/conf-enabled
```

Comment:
Was this a bug?

Reply:
I think there was a really old server with a version of monit that didn't come with this dir when it was first installed?